assets/js/base/components/cart-checkout/order-summary/order-summary-item.js

```diff
@@ -5,6 +5,7 @@ import { __, sprintf } from '@wordpress/i18n';
 import { getCurrency } from '@woocommerce/base-utils';
 import Label from '@woocommerce/base-components/label';
 import {
+	ProductBackorderNotification,
```

Comment:
`Notification` doesn't seem quite right for this – to me that suggests this is a notice or snackbar notification. I'd go with `badge` for consistency, e.g. `ProductBackorderBadge`. Minor nitpick - your call :)

Reply:
Good point. Changed in 4c26498c0f063763e0592ccb698cf08a2280eea1.